src/main/kotlin/platform/mixin/expression/MEExpressionCompletionUtil.kt

```diff
@@ -1271,13 +1275,25 @@ object MEExpressionCompletionUtil {
                     val fixedNewArrayExpr = factory.createExpression("new ?[?]") as MENewExpression
                     fixedNewArrayExpr.type!!.replace(type)
                     variants += fixedNewArrayExpr
+
+                    val arrayLitExpr = factory.createExpression("new ?[]{?}") as MENewExpression
+                    arrayLitExpr.type!!.replace(type)
+                    variants += arrayLitExpr
                 }
             }
+            is MESuperCallExpression -> {
+                // Might be missing its parentheses
+                val callExpr = factory.createExpression("super.?()") as MESuperCallExpression
+                expression.memberName?.let { callExpr.memberName!!.replace(it) }
+                variants += callExpr
+            }
         }
 
         return variants
     }
 
+    private fun createUniqueLookup(text: String) = LookupElementBuilder.create(Any(), text)
```

Comment:
I'm a bit concerned that this looks a bit like a hack. Is the first parameter used for anything important?

Reply:
It's used by custom sorters but we don't have any of those. Otherwise it's just used to determine the uniqueness.